src/main/java/org/cloudburstmc/api/Server.java

```diff
@@ -20,32 +19,96 @@
     String BROADCAST_CHANNEL_ADMINISTRATIVE = "cloudburst.broadcast.admin";
     String BROADCAST_CHANNEL_USERS = "cloudburst.broadcast.user";
 
+    /**
+     * Gets the name of this server implementation.
+     *
+     * @return name of this server
+     */
     String getName();
 
-    String getVersion();
-
+    /**
+     * Gets the version of this server implementation.
+     *
+     * @return version of this server
+     */
     String getImplementationVersion();
 
-    void shutdown();
-
-    boolean isRunning();
+    /**
+     * Returns the Minecraft version supported by this server implementation.
+     *
+     * @return the supported Minecraft version
+     */
+    String getMinecraftVersion();
+
+    /**
+     * Returns the current version of this API.
+     *
+     * @return the API version
+     */
+    String getApiVersion();
 
+    /**
+     * Gets a collection of players currently online on this server.
+     *
+     * @return a collection of online players
+     */
+    Collection<? extends Player> getOnlinePlayers();
+
+    /**
+     * Find a player by their username.
+     *
+     * @param name the name of the player
+     * @return an instance of that player if online
+     */
+    Player findPlayer(String name);
```

Comment:
I changed this to `findPlayer` as `find` better conveys the possibility of not finding a player (if they aren't online)